src/runtime-rs/crates/runtimes/common/src/types/mod.rs

```diff
@@ -152,6 +178,50 @@ pub struct SandboxConfig {
     pub state: runtime_spec::State,
 }
 
+#[derive(Clone, Debug)]
+pub struct SandboxID {
+    pub sandbox_id: String,
+}
+
+#[derive(Clone, Debug)]
+pub struct StartSandboxInfo {
+    pub pid: u32,
+    pub create_time: Option<std::time::SystemTime>,
+}
+
+#[derive(Clone, Debug)]
+pub struct PlatformInfo {
+    pub os: String,
+    pub architecture: String,
+}
+
+#[derive(Clone, Debug)]
+pub struct StopSandboxRequeset {
```

Comment:
It would be nice to fix the typo here and elsewhere by searching for "Requeset" and replacing it with "Request".

Reply:
Done